serverless/cluster-sizing-requirements.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// serverless/installing-openshift-serverless.adoc
+
+
+[id="cluster-size-requirements_context"]
+= Cluster size requirements
+include::modules/common-attributes.adoc[]
+:context: cluster-size-requirements
+
+The cluster must be sized appropriately to ensure that OpenShift Serverless can run correctly. You can use the MachineSet API to manually scale your cluster up to the desired size.
+
+An OpenShift cluster with 10 CPUs and 40 GB memory is the minimum requirement for getting started with your first serverless application. This usually means you will have to scale up one of the default MachineSets by two additional machines.
+
```

Comment:
I think this section is now missing some parts of the updated installation document:

Note: The requirements are greatly depending on the deployed applications. By default, each pod requests ~400m of CPU and the recommendation is based on that. With the given recommendation, such an application can scale up to 10 replicas. Lowering the actual CPU request of the application further pushes that boundary.

For more advanced full-fledged use-cases (for example making use of Monitoring, Metering and Tracing) you’ll need to deploy more resources according to the respective documentation. A good starting point are 24 vCPUs and 96GB of memory.

For OpenShift Logging, there are specific CPU and memory requirements. The Elasticsearch node needs 16G of memory for both memory requests and CPU limits. You might need to add additional nodes to the OpenShift Container Platform cluster to run with the recommended or higher memory. When running the OpenShift Logging on a cluster in AWS the recommended instance type is m4.2xlarge (8 vCPUs, 32 GiB Mem).